groups/style.css

```diff
@@ -100,6 +100,7 @@ body {
   justify-content: center;
   align-items: center;
   flex-direction: column;
+  min-height: 98vh;
```

Comment:
Where 98 come from is UI is working on all the different size

Reply:
Fixed it.